core/src/main/java/org/apache/accumulo/core/clientImpl/TabletServerBatchWriter.java

```diff
@@ -209,20 +210,17 @@ public TabletServerBatchWriter(ClientContext context, BatchWriterConfig config)
     this.writer = new MutationWriter(config.getMaxWriteThreads());
 
     if (this.maxLatency != Long.MAX_VALUE) {
-      jtimer.schedule(new TimerTask() {
-        @Override
-        public void run() {
-          try {
-            synchronized (TabletServerBatchWriter.this) {
-              if ((System.currentTimeMillis() - lastProcessingStartTime)
-                  > TabletServerBatchWriter.this.maxLatency)
-                startProcessing();
-            }
-          } catch (Throwable t) {
-            updateUnknownErrors("Max latency task failed " + t.getMessage(), t);
+      executor.scheduleWithFixedDelay(Threads.createNamedRunnable("latency timer", () -> {
```

Comment:
I like the fact that previously the timer thread would have `BatchWriter` in its name. Only `latency timer` makes it just a little harder to figure out when looking at a thread dump, IMO.
```suggestion
      executor.scheduleWithFixedDelay(Threads.createNamedRunnable("BatchWriterLatencyTimer", () -> {
```

Reply:
That's a good catch, I can change the thread name.